### PR TITLE
Add VEGAS Placeholders

### PR DESCRIPTION
**Pull request by gomezzz**

### CODE REVIEW

File: torchquad/integration/vegas.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+            torch.random.manual_seed(seed)
+
+        self.map = VEGASMap()
+        self.strat = VEGASStratification()
+
+        logger.debug("Running grid warmup")
+        self._improve_grid()
+
+        logger.debug("Running VEGAS Iterations")
+        self.results = []  # contains integrations results per iteration
```
Comment (htoftevaag) [inline torchquad/integration/vegas.py:79]: "contains integrations results per iteration" -> contains integration results per iteration
or 
-> contains integration's results per iteration
or
-> contains integrations' results per iteration

File: torchquad/integration/vegas_map.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import torch
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class VEGASMap:
+    """The map used for vegas enhanced. Refer to https://arxiv.org/abs/2009.05112.
```
Comment (htoftevaag) [inline torchquad/integration/vegas_map.py:9]: vegas -> VEGAS

File: torchquad/integration/vegas_stratification.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import torch
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class VEGASStratification:
+    """The stratification used for vegas enhanced. Refer to https://arxiv.org/abs/2009.05112.
```
Comment (htoftevaag) [inline torchquad/integration/vegas_stratification.py:9]: Same, vegas -> VEGAS

**Review by htoftevaag — CHANGES_REQUESTED** [review]
Please see my comments.

**Comment by gomezzz** [comment]
I have added the requested changes.

File: torchquad/integration/vegas.py [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+import torch
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class VEGAS(BaseIntegrator):
+    """Vegas Enhanced in torch. Refer to https://arxiv.org/abs/2009.05112.
```
Comment (htoftevaag) [inline torchquad/integration/vegas.py:15]: Sorry for not noticing this the first time, but shouldn't it be `VEGAS Enhanced` here as well?

**Review by htoftevaag — CHANGES_REQUESTED** [review]

**Pull request merged**